packages/google_maps_flutter/lib/src/bitmap.dart

```diff
@@ -59,6 +59,13 @@ class BitmapDescriptor {
     AssetBundle bundle,
     String package,
   }) async {
+    if (configuration.devicePixelRatio != null) {
+      return BitmapDescriptor._(<dynamic>[
+        'fromAssetImage',
+        assetName,
+        configuration.devicePixelRatio,
+      ]);
+    }
```

Comment:
line 76 can be `configuration.devicePixelRatio ?? assetBundleImageKey.scale`

Reply:
That would invoke unnecessary calls to 
```
    final AssetImage assetImage =
        AssetImage(assetName, package: package, bundle: bundle);
    final AssetBundleImageKey assetBundleImageKey =
        await assetImage.obtainKey(configuration);
```
I don't think we need to waste CPU time and battery doing that when scaling is already provided.

Reply:
Makes sense.